### ISSUE TITLE
Uncaught Error: Invalid rootpath

### ISSUE THREAD

**Issue by VictorVation** [facebook/nuclide#1060]
1. Open "Source Control" tab 
1. Change some files in the repo.
2. Right click a file under "Uncommitted Changes"
3. Context menu doesn't appear, throws error.

**Atom**: 1.15.0 x64
**Electron**: 1.3.13
**OS**: Mac OS X 10.12.3
**Thrown From**: [nuclide](https://github.com/facebook/nuclide) package 0.210.52418942

### Stack Trace

Uncaught Error: Invalid rootpath

```
At /opt/facebook/nuclide/0.210.52418942/nuclide/node_modules/rxjs/bundles/Rx.min.js:33

Error: Invalid rootpath
  at MultiRootChangedFilesView._getStatusCodeForFile (/opt/facebook/nuclide/0.210.52418942/nuclide/pkg/nuclide-ui/MultiRootChangedFilesView.js:138:13)
  at Object.shouldDisplay (/opt/facebook/nuclide/0.210.52418942/nuclide/pkg/nuclide-ui/MultiRootChangedFilesView.js:62:23)
  at ContextMenuManager.module.exports.ContextMenuManager.cloneItemForEvent (/app.asar/src/context-menu-manager.js:129:19)
  at ContextMenuManager.module.exports.ContextMenuManager.templateForEvent (/app.asar/src/context-menu-manager.js:87:33)
  at ContextMenuManager.module.exports.ContextMenuManager.showForEvent (/app.asar/src/context-menu-manager.js:187:27)
  at a.sub.nextAnimationFrame.repeat.last.subscribe [as _next] (/opt/facebook/nuclide/0.210.52418942/nuclide/pkg/nuclide-file-tree/lib/main.js:132:22)
  at a.__tryOrUnsub (/opt/facebook/nuclide/0.210.52418942/nuclide/node_modules/rxjs/bundles/Rx.min.js:33:10)
  at a.next (/opt/facebook/nuclide/0.210.52418942/nuclide/node_modules/rxjs/bundles/Rx.min.js:31:454)
  at a._next (/opt/facebook/nuclide/0.210.52418942/nuclide/node_modules/rxjs/bundles/Rx.min.js:30:320)
  at a.next (/opt/facebook/nuclide/0.210.52418942/nuclide/node_modules/rxjs/bundles/Rx.min.js:29:506)
  at a._complete (/opt/facebook/nuclide/0.210.52418942/nuclide/node_modules/rxjs/bundles/Rx.min.js:178:191)
  at a.complete (/opt/facebook/nuclide/0.210.52418942/nuclide/node_modules/rxjs/bundles/Rx.min.js:30:155)
  at a._complete (/opt/facebook/nuclide/0.210.52418942/nuclide/node_modules/rxjs/bundles/Rx.min.js:30:456)
  at a.complete (/opt/facebook/nuclide/0.210.52418942/nuclide/node_modules/rxjs/bundles/Rx.min.js:30:155)
  at a.complete (/opt/facebook/nuclide/0.210.52418942/nuclide/node_modules/rxjs/bundles/Rx.min.js:195:492)
  at requestAnimationFrame (/opt/facebook/nuclide/0.210.52418942/nuclide/pkg/commons-node/observable.js:283:14)

```

### Commands

```
  2x -1:09.4.0 fb-code-snippet-view:toggle (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
     -1:08.9.0 core:save (input.hidden-input)
  3x -1:08.9.0 fb-code-snippet-view:toggle (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
     -1:08.5.0 core:backspace (input.hidden-input)
  2x -1:08.5.0 fb-code-snippet-view:toggle (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
     -1:08.3.0 core:save (input.hidden-input)
 15x -1:08.3.0 fb-code-snippet-view:toggle (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
     -0:12.9.0 find-and-replace:show (input.hidden-input)
  2x -0:12.3.0 core:confirm (input.hidden-input)
     -0:11.9.0 editor:consolidate-selections (input.hidden-input)
     -0:11.9.0 core:cancel (input.hidden-input)
     -0:11.7.0 vim-mode-plus:activate-normal-mode (input.hidden-input)
     -0:11 diff-view-navigator:toggle (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
     -0:10.8.0 nuclide-move-item-to-available-pane:left (atom-text-editor.editor.vim-mode-plus.normal-mode)
     -0:09.7.0 fb-code-snippet-view:toggle (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
  3x -0:08.2.0 diff-view-navigator:toggle (atom-workspace.workspace.scrollbars-visible-always.theme-one-dark-syntax.theme-one-dark-ui)
```

### Non-Core Packages

```
atom-beautify 0.29.17 
atom-clock 0.1.5 
copy-path 0.4.1 
ex-mode 0.13.1 
haskell-grammar 0.4.0 
highlight-selected 0.12.0 
language-babel 2.55.6 
language-graphql 0.8.0 
language-ini 1.18.0 
language-lua 0.9.11 
language-ocaml 1.1.2 
language-swift 0.5.0 
language-thrift 1.0.2 
last-cursor-position 0.9.0 
merge-conflicts 1.4.4 
nuclide 0.210.52418942 
nuclide-format-js 0.0.36 
set-syntax 0.3.2 
sort-lines 0.14.0 
tool-bar 1.0.1 
vim-mode-plus 0.82.3 
vim-mode-plus-ex-mode 0.9.0 
```

**Comment by aadisriram** [comment]
I'm able to repro this, looks like the click event is getting funneled down to the Icon but the context menu shows up if you click anywhere else but the icon itself. I'm looking into pushing a fix for this soon.

**Comment by VictorVation** [comment]
@aadisriram Thanks for the quick response! Clicking the filename rather than the icon works as a workaround for now 😄

**Comment by aadisriram** [comment]
Fixed it in v0.211

**Comment by akondrasev** [comment]
still repro in v0.214.0
